crates/utils/src/tracing/grpc.rs

```diff
@@ -46,6 +46,7 @@ fn rpc_trace_fn<T>(request: &http::Request<T>) -> tracing::Span {
         Some("CheckNullifiers") => rpc_span!("rpc.rpc", "CheckNullifiers"),
         Some("CheckNullifiersByPrefix") => rpc_span!("rpc.rpc", "CheckNullifiersByPrefix"),
         Some("GetBlockHeaderByNumber") => rpc_span!("rpc.rpc", "GetBlockHeaderByNumber"),
+        Some("SyncAccountVault") => rpc_span!("rpc.rpc", "SyncAccountVault"),
```

Comment:
Related to https://github.com/0xMiden/miden-node/pull/1140: should we also add an entry here for `SyncAccountStorageMaps`? (I didn't see it in the other PR).